src/app/client_sdk/tests/test_signatures.ml

```diff
@@ -0,0 +1,107 @@
+(* test_signatures.ml -- generate signatures for some transactions,
+    for comparison against signatures generated in client SDK
+ *)
+
+[%%import
+"/src/config.mlh"]
+
+open Core_kernel
+
+[%%ifdef
+consensus_mechanism]
+
+open Snark_params.Tick
+
+[%%else]
+
+open Snark_params_nonconsensus
+module Coda_base = Coda_base_nonconsensus
+module Signature_lib = Signature_lib_nonconsensus
+module Currency = Currency_nonconsensus.Currency
+module Coda_numbers = Coda_numbers_nonconsensus.Coda_numbers
+
+[%%endif]
+
+open Coda_base
+open Signature_lib
+
+(* signer *)
+let keypair =
+  let private_key =
+    Private_key.of_base58_check_exn
+      "6BnSDyt3FKhJSt5oDk1HHeM5J8uKSnp7eaSYndj53y7g7oYzUEhHFrkpk6po4XfNFyjtoJK4ovVHvmCgdUqXVEfTXoAC1CNpaGLAKtu7ah9i4dTi3FtcoKpZhtiTGrRQkEN6Q95cb39Kp"
```

Comment:
will these need to change if the key formats change? Maybe we can centralize a "dummy" keypair that we can use in places like this so we only need to update this in one spot

Reply:
Yes, they'd need to change, but we'd still need to make the change in `test_signatures.js`.

There are actually 3 keypairs involved here.

I suppose we could have a JSON file with some dummy keypairs. Can we defer that to another PR (this one is big enough). I can open an issue.